lib/beaker/hypervisor/openstack.rb

```diff
@@ -133,7 +133,7 @@ def provision
         @vms << vm
 
         #enable root if user is not root
-        enable_root_on_hosts()
+        enable_root_on_host(host)
```

Comment:
@fiddyspence, taking a look at this now that I'm more clear on what the change is, I'm realizing that this line is really just using the wrong method.  

`enable_root_on_hosts` shouldn't actually disappear (it might be needed for other things later), but this usage should actually just skip to using `enable_root`, and `enable_root_on_host` shouldn't exist, since it doesn't do anything more than pass the host to `enable_root`.

My apologies on coming to the main issue late like this, and asking you to do different things because I didn't understand what was going on here.